CIP-dapp-bridge-rework/README.md

```diff
@@ -0,0 +1,1169 @@
+
+#### `HexString`
+
+Byte data, converted to hex and stored as a string value. For example, the hash of something. Note, this is not the same as cbor<T>, although there is no difference in representation between them, there is a difference at the level of context and use cases!<br><br>
+
+#### `base58`
+
+A base58-encoded hexadecimal value. Typically used for Byron addresses.<br><br>
+
+#### `AddressString`
+
+A string representing an address in either bech32 or base58 format. Not CBOR!<br><br>
+
+#### `LocalString`
+
+_**Array**_ of objects describing a set of localizations of the same text, intended for the wallet user.
+
+```ts
+type LocalString = {
+  local: string,
+  body: string
+}[];
+```
+
+* **`local`**
+  
+  A string containing the locale code in the format [ISO-639.1](https://en.wikipedia.org/wiki/ISO_639-1).
+  
+  > **Note:** The DApp is not obliged to provide messages in languages it does not support.
+  
+* **`body`**
+
+  A string containing the localized text of the message. It is important that all `body` within one `LocalString` contain the same information. <br><br>
+
+The logic for working with `LocalString` is as follows:
+
+1. If a `LocalString` consisting of only one element is passed to the wallet, it is displayed "as is".
+2. If a `LocalString` consisting of several elements is passed to the wallet, the wallet should choose the one whose `local` is most suitable for the user.
+3. If the `LocalString` does not contain a localization suitable for the user, the wallet may choose any other localization it considers "default". In most cases, this will likely be "en". <br><br>
+
+
+#### `AccountEra`
+
+A string indicating whether the account belongs to the `Byron` or `Shelley` eras of Cardano.
+
+```ts
+type AccountEra = "Byron" | "Shelley";
+```
+
+#### `AccountId`
+
+An arbitrary string that serves as the identifier of the wallet account connected to the DApp, formed by the wallet according to its own rules. The main point is that the wallet understands what is required of it when it receives this identifier in a request.
+
+```ts
+type AccountId = string;
+```
+
+#### `NativeToken`
+
+A structure describing a `native token` in the Cardano network.
+
+```ts
+type NativeToken = {
+    policyId: HexString,
+    name: HexString | null,
+    amount: BigIntString
+};
+
+```
+
+* **`policyId`**
+
+  Contains the [hexadecimal](#hexstring) representation of the asset's `policyID`.<br><br>
+
+* **`name`**
+
+  Contains the [hexadecimal](#hexstring) representation of the asset's `name`. If absent, the value should be set to `null`.<br><br>
+
+* **`amount`**
+
+  Contains the quantity of the asset [without a decimal part](#bigintstring). That is, if we want to describe 1 token with a decimal part of 6 digits, the amount should be set to 1,000,000, i.e., 1 * 10^6. Setting the value to 1 would be equivalent to 0.000001!<br><br>
```

Comment:
This is an example of what I was referring to above: loose content which would be better formatted as bullet lists or tables... especially because you're not using headings that would create deep links to these entries.

This pattern is repeated several times throughout the document, and unless addressed it will affect readability by making the CIP look longer & more complicated than it really is.

Reply:
"Should I remove the `brs` only, or attach the content of each item to its header in one row?

Reply:
:+1: I think without the `<br>` elements Markup wiil create a white spacing consistent with other documents.... in fact in hindsight that's what was creating the extra line spacing rather than the carriage returns as I suggested above (so resolving that one).

Yes I'd also look to see how it looks as a table with a row for each parameter, if you're also interested, which I think would make it a better match for other CIPs & computer documentation in general.

Reply:
>Yes I'd also look to see how it looks as a table with a row for each parameter, if you're also interested, which I think would make it a better match for other CIPs & computer documentation in general.

I'll check this approach. But there are many variations of docs styles. My reference in most cases was the [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object).